### PR TITLE
Add Super Paper Mario preset

### PR DESCRIPTION
**Pull request by SeekyCt**

### CODE REVIEW

File: frontend/src/components/compiler/PresetSelect.tsx [hunk 1]
```diff
@@ -85,6 +85,16 @@ export const PRESETS = [
         compiler: "mwcc_233_163e",
         opts: "-O4,p -nodefaults -fp hard -Cpp_exceptions off -enum int -fp_contract on -inline auto",
     },
+    {
+        name: "Super Paper Mario (DOL)",
+        compiler: "mwcc_41_60831",
+        opts: "-lang=c99 -enc SJIS -fp hard -O4 -use_lmw_stmw on -str pool -rostr -inline all -sdata 4 -sdata2 4"
```
Comment (nanaian) [inline frontend/src/components/compiler/PresetSelect.tsx:91]:
```suggestion
        opts: "-lang=c99 -enc SJIS -fp hard -O4 -use_lmw_stmw on -str pool -rostr -inline all -sdata 4 -sdata2 4",
```

File: frontend/src/components/compiler/PresetSelect.tsx [hunk 2]
```diff
@@ -85,6 +85,16 @@ export const PRESETS = [
...
+    },
+    {
+        name: "Super Paper Mario (REL)",
+        compiler: "mwcc_41_60831",
+        opts: "-lang=c99 -enc SJIS -fp hard -O4 -use_lmw_stmw on -str pool -rostr -ipa file -sdata 0 -sdata2 0 -pool off -ordered-fp-compares"
```
Comment (nanaian) [inline frontend/src/components/compiler/PresetSelect.tsx:96]:
```suggestion
        opts: "-lang=c99 -enc SJIS -fp hard -O4 -use_lmw_stmw on -str pool -rostr -ipa file -sdata 0 -sdata2 0 -pool off -ordered-fp-compares",
```

**Review by nanaian — CHANGES_REQUESTED** [review]
As per eslint

File: frontend/src/components/compiler/PresetSelect.tsx (hunk 2 above)
Comment (SeekyCt) [reply frontend/src/components/compiler/PresetSelect.tsx:96]: Apparently I'm blind, I manually checked to see if the other presets did that after I noticed there was one after the final } and somehow thought they didn't lol, will fix quickly

**Review by nanaian — APPROVED** [review]

**Pull request merged**